src/cloudabi-run/cloudabi-run.1

```diff
@@ -110,6 +110,10 @@ or the special values
 .Dq Li stdout
 and
 .Dq Li stderr .
+.It Cm "tag:nuxi.nl,2015:cloudabi/executable: int"
```

Comment:
Could you remove the `: int` part? The man page uses that notation to specify that the tag needs to be followed by an integer. In this case, `!executable` should be followed by nothing.